pkgs/applications/networking/irc/iim/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, stdenv }:
+stdenv.mkDerivation {
+  name = "iim";
+
+  src = builtins.fetchGit {
```

Comment:
my mistake, this should be fetchFromGithub (and added to the inputs at the top

Reply:
@tomberek What does "added to the inputs at the top" mean here? Apologies for my confusion

Reply:
You should use fetchFromGitHub which...